02-express-mongo-crud/intro-express/22-23/README.md

```diff
@@ -0,0 +1,83 @@
+#Intro to Express
+
+| Objectives |
+| :--- |
+| Explain the request response cycle |
+| Use npm (node package manager) to initialize a node project |
+| Write a local Node.js web server with Express |
+| Add EJS tempting engine to an Express app |
+
+### Motivation (Why?)
+
+Express is a cutting-edge, unopinionated, server-side JavaScript framework that runs on a Node.js server. It is a very, very popular and trending framework with a bevy of modules you can add to it.
+
+### What is Node?
+- Node.js is a webserver that operates on the V8 Google Chrome JavaScript runtime, allowing you to write server-side code in JavaScript.
```

Comment:
Node.js itself isn't really a webserver is it?  It's just a quoting node.js.org/en  "a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. "